# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: b021b326fa47
# Full Hash: b021b326fa47eb57e14f858078ab5daaa6cd2274
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-06-20 03:53:45
# Regressor Bug: 1839070
# File Overlap Count: 6
# Description:
#   Bug 1839070 - Emit a start point if close is called immediately after move_to. r=jrmuizel
#   
#   wpf-gpu-raster would blindly modify the last point type in a path when call was closed, even
#   if move_to preceded it directly and outputted no actual point type. In this case, we need to
#   ensure that we emit a start point if close is called immediately after move_to.
# ==============================================================================

diff -r 4c4b0b9db374 -r b021b326fa47 .cargo/config.in
--- a/.cargo/config.in	Mon Jun 19 19:34:31 2023 +0000
+++ b/.cargo/config.in	Mon Jun 19 19:54:24 2023 +0000
@@ -10,9 +10,9 @@
 rev = "07d3c25322518f294300e96246e09b95e118555d"
 replace-with = "vendored-sources"
 
-[source."git+https://github.com/FirefoxGraphics/wpf-gpu-raster?rev=a6514854d4518b02f2805719ff6cd74dae7dfde6"]
+[source."git+https://github.com/FirefoxGraphics/wpf-gpu-raster?rev=112a2140c433af06938c663fc6777e2bf17bb6ce"]
 git = "https://github.com/FirefoxGraphics/wpf-gpu-raster"
-rev = "a6514854d4518b02f2805719ff6cd74dae7dfde6"
+rev = "112a2140c433af06938c663fc6777e2bf17bb6ce"
 replace-with = "vendored-sources"
 
 [source."git+https://github.com/chris-zen/coremidi.git?rev=fc68464b5445caf111e41f643a2e69ccce0b4f83"]